scm-ui/ui-webapp/src/containers/InitializationAdminAccountStep.tsx

```diff
@@ -0,0 +1,227 @@
+  const [startupKey, setStartupKey] = useState("");
+  const [userName, setUserName] = useState("scmadmin");
+  const [displayName, setDisplayName] = useState("SCM Administrator");
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [passwordConfirmation, setPasswordConfirmation] = useState("");
+  const [userNameValidationError, setUserNameValidationError] = useState(false);
+  const [displayNameValidationError, setDisplayNameValidationError] = useState(false);
+  const [mailValidationError, setMailValidationError] = useState(false);
+  const [passwordValidationError, setPasswordValidationError] = useState(false);
+  const [passwordConfirmationValidationError, setPasswordConfirmationValidationError] = useState(false);
+
+  const { create, isLoading, error, isCreated } = useCreateAdmin((data._links.initialAdminUser as Link).href);
+
+  useEffect(() => {
+    if (isCreated) {
+      window.location.reload(false);
+    }
+  }, [isCreated]);
+
+  const validateAndSetUserName = (newUserName: string) => {
+    setUserNameValidationError(!validation.isNameValid(newUserName));
+    setUserName(newUserName);
+  };
+
+  const validateAndSetDisplayName = (newDisplayName: string) => {
+    setDisplayNameValidationError(!isDisplayNameValid(newDisplayName));
+    setDisplayName(newDisplayName);
+  };
+
+  const validateAndSetEmail = (newEmail: string) => {
+    setMailValidationError(!validation.isMailValid(newEmail));
+    setEmail(newEmail);
+  };
+
+  const validateAndSetPassword = (newPassword: string) => {
+    setPasswordValidationError(!isPasswordValid(newPassword));
+    setPassword(newPassword);
+  };
+
+  const validateAndSetPasswordConfirmation = (newPasswordConfirmation: string) => {
+    setPasswordConfirmationValidationError(newPasswordConfirmation !== password);
+    setPasswordConfirmation(newPasswordConfirmation);
+  };
+
+  const handleSubmit = (event: FormEvent) => {
+    event.preventDefault();
+    create({
+      startupToken: startupKey,
+      userName,
+      displayName,
+      email,
+      password,
+      passwordConfirmation,
+    });
+  };
+
+  let errorComponent;
+  if (error) {
+    if (error.message === "Forbidden") {
+      errorComponent = <ErrorNotification error={new Error(t("error.forbidden"))} />;
+    } else {
+      errorComponent = <ErrorNotification error={error} />;
+    }
+  }
+
+  const formValid =
+    !!(startupKey && userName && displayName && email && password && passwordConfirmation) &&
+    !(
+      userNameValidationError ||
+      displayNameValidationError ||
+      mailValidationError ||
+      passwordValidationError ||
+      passwordConfirmationValidationError
+    );
+  const component = (
+    <div className="column is-8 box  has-background-white-ter">
+      <form onSubmit={handleSubmit}>
+        <h3 className="title">{t("title")}</h3>
+        <h4 className="subtitle">{t("adminStep.title")}</h4>
+        {errorComponent}
```

Comment:
Just my opinion, but i think i would like to show the error right above the submit button.